lib/metasploit/framework/password_crackers/hashcat/formatter.rb

```diff
@@ -1,3 +1,24 @@
+# This method takes a string which is likely base64 encoded
+# however, there is an arbitrary amount of = missing from the end
+# so we attempt to add = until we are able to decode it
+#
+# @param str [String] the base64-ish string
+# @return [String] the corrected string
+private
```

Comment:
I don't think this keyword does anything unless it's in a class or module

Reply:
that's what I thought as well, however I wasn't sure how else to resolve the previous comment review comment